xapian-applications/omega/omega.cc

```diff
@@ -360,6 +365,10 @@ try {
     for (auto i = begin; i != end; ++i) {
 	const string & v = i->second;
 	if (!v.empty()) {
+	    if (i->first[CONST_STRLEN("START.")] < '0' ||
+		i->first[CONST_STRLEN("START.")] > '9') {
+		throw "START slot value must be >= 0\n";
+	    }
 	    Xapian::valueno slot = atoi(i->first.c_str() +
 					CONST_STRLEN("START."));
```

Comment:
Was there a reason to only check for a leading digit and then use `atoi()` here (and in the two cases below)?

That allows arbitrary text after the slot number, but the documentation doesn't seem to say that arbitrary trailing text is allowed there, and I can't really see a reason why it's useful in this case.

Reply:
oh sorry, my mistake. will fix it.